docs/about.md

```diff
@@ -6,39 +6,135 @@
 
 <span class="lead">MarkBind is a project **based in [National University of Singapore, School of Computing](http://www.comp.nus.edu.sg/)**, and is funded by an education grant from [NUS Center for Development of Teaching and Learning](https://nus.edu.sg/cdtl).</span>
 
-The **project team**:
 
-* [**Aaron Chong Jun Hao**](https://github.com/acjh): _Project Mentor_ since Jul 2018, _Team Lead_ for Aug 2017 - Jun 2018
-* [**Ang Ze Yu**](https://www.github.com/ang-zeyu): _Project Member_ since Jan 2020
-* [**Chng Zhi Xuan**](https://github.com/Chng-Zhi-Xuan): _Project Member_ since May 2018
-* [**Chua Yun Zhi Nicholas**](https://github.com/nicholaschuayunzhi): _Project Member_ since Jan 2018
+## Contributors ✨
+
+Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/docs/en/emoji-key)):
+
+<div class="table-responsive">
```

Comment:
Added to make it responsive when it's shown on the about-us page.
- added outside of the comments below to prevent it from being overridden by all-contributor updates